src/gui/editors/PianoRoll.cpp

```diff
@@ -3522,6 +3522,23 @@ void PianoRoll::finishRecordNote(const Note & n )
 						it->key(), it->getVolume(),
 						it->getPanning() );
 				n1.quantizeLength( quantization() );
+
+				//Get selected chordModel
+				const InstrumentFunctionNoteStacking::Chord & chord = InstrumentFunctionNoteStacking::ChordTable::getInstance()
+						.getChordByName( m_chordModel.currentText() );
+
+					if( ! chord.isEmpty() )
+					{
+						// if a chord is selected, create following notes in chord mode
+						for( int i = 1; i < chord.size(); i++ )
+						{
+							Note new_note( n.length(), it->pos(), it->key() + chord[i] );
```

Comment:
We prefer `camelCase` to `snake_case`, though it's not in the coding convention.